app/shell/Status/Status.module.scss

```
@@ -1,21 +1,33 @@
 @import 'config/bundle';
 
+.nodeStatus,
 .body {
     composes: body4 from global;
     display: flex;
     align-items: center;
     justify-content: center;
     margin: 20px;
```

For me, the spacing between statuses looks a little off. Following suggestion looked better to me.

```suggestion
    margin: 10px 20px;
```

What do you think?

---

I agree.